site/docs/spec.md

```diff
@@ -124,6 +126,8 @@ A **`list`** is a collection of values with some element type. The element field
 
 A **`map`** is a collection of key-value pairs with a key type and a value type. Both the key field and value field each have an integer id that is unique in the table schema. Map keys are required and map values can be either optional or required. Both map keys and map values may be any type, including nested types.
 
+For semantics of default value support for fields of Nested Types, see Appendix E. 
```

Comment:
I feel that spec behavior of default values is worth explained directly in the spec rather than as part of an appendix.